cms/templates/js/course-outline.underscore

```diff
@@ -235,6 +235,25 @@ if (is_proctored_exam) {
                         <% } %>
                     </p>
                 </div>
+                <% if (course.get('self_paced') && course.get('is_custom_relative_dates_active') && xblockInfo.get('relative_weeks_due')) { %>
```

Comment:
These vars were added in some previous PR I gather.

Reply:
yup! [here's the PR if curious](https://github.com/edx/edx-platform/pull/28175)